### PR TITLE
Refresh /organisations

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Note: Some assets are missing so I have added in 'placeholders' for the time being**
- Rebuild page ready for the 22.04 refresh as per [copydoc](https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit#)

## QA

- Go to https://ubuntu-com-11492.demos.haus/desktop/organisations and compare against [copydoc](https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4960

### CODE REVIEW

File: templates/desktop/organisations.html [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
     <div class="row u-equal-height u-vertically-center">
       <div class="col-6">
-        <h1>Ubuntu in organisations</h1>
+        <h1>Bring Ubuntu to your <br> organisation</h1>
         <p>
-          Ubuntu is adopted by small and large teams alike because it is easy to use, highly secure, has a low cost of ownership and a huge range of apps.
+          Ubuntu Desktop combines enterprise-grade support, security <br>and functionality with the best of open source.
         </p>
         <p><a class="p-button--positive js-invoke-modal" href="/desktop/contact-us" data-testid="interactive-form-link">Contact us</a></p>
         <p><a href="/advantage" class="p-link--inverted">Buy Ubuntu Advantage&nbsp;&rsaquo;</a></p>
```
Comment (carkod) [inline templates/desktop/organisations.html:17]: This link is https://ubuntu.com/advantage/subscribe

File: templates/desktop/organisations.html [hunk 2]
```diff
@@ -20,296 +20,782 @@ <h1>Ubuntu in organisations</h1>
...
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/be731535-Logo-libreoffice.svg",
+    <h2>Ubuntu delivers the best of open source</h2>
+    <p>Drive innovation in high-growth areas such as data science and machine learning with support for the latest libraries and apps  in addition to critical workflow tools like <a href="https://multipass.run/">Multipass</a>, <a href="https://jaas.ai/">Juju</a>, <a href="/lxd">LXD</a>, <a href="https://microk8s.io/">MicroK8s</a>.</p>
+    <div class="p-logo-section">
+      <div class="p-logo-section__items">
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/57dd2115-TensorFlow-logo.svg",
```
Comment (carkod) [inline templates/desktop/organisations.html:238]: Arm logo https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit?disco=AAAAWjOJjN8

Also does order matter?

**Comment by local-optimum** [comment]
Feedback:
- The OEMs (lenovo, HP, Dell) should be noticeabley separate from the component vendors in that logo cloud
- MicroK8s logo looks very small
- I added extra info to one of the UA support rows in the bottom table
- The banner image needs to change, it's very out of date and low end device.

File: templates/desktop/organisations.html [hunk 3]
```diff
@@ -20,296 +20,782 @@ <h1>Ubuntu in organisations</h1>
...
+          <td>Cost per seat per year</td>
+          <td class="u-align--center"><strong>$25</strong></td>
+          <td class="u-align--center"><strong>$150</strong></td>
+          <td class="u-align--center"><strong>$300</strong></td>
+        </tr>
+        <tr>
+          <td>Phone and ticket support</td>
+          <td></td>
+          <td class="u-align--center">24/5</td>
+          <td class="u-align--center">24/7</td>
```
Comment (carkod) [inline templates/desktop/organisations.html:667]: Copy doc has been updated with new info https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit?disco=AAAAXq0BaRQ

**Review by carkod — APPROVED** [review]
Overall looks good, but there's been some more copy update added.

File: templates/desktop/organisations.html (hunk 2 above)
Comment (petesfrench) [reply templates/desktop/organisations.html:238]: The order in the doc no, I've ordered it like this to try and balance the different sized logos

**Comment by local-optimum** [comment]
@petesfrench I think this is pretty much there bar the placeholders, let's leave the micoK8s for now.

I feel like there needs to be a consistent middle ground for the AI/ML flow diagram and the 'write once deploy anywhere' flow diagram. They should be the same size, something in between both of their current sizes.

**Pull request merged**